### PR TITLE
Display a JEE assessment and generate a draft plan from the goals

### PR DESCRIPTION
**Pull request by savannidgerinel**

### CODE REVIEW

File: app/lib/goal_form.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      params.keys.reduce({}) do |acc, key|
+        unless key.start_with?('jee1_') || key.start_with?('jee2_') ||
+               key.start_with?('spar_')
+          next acc
+        end
+        next acc if key.end_with?('_goal')
+        score = params[key].to_i
+        goal = params["#{key}_goal"].to_i
+
+        puts "key #{key} not found in crosswalk" unless crosswalk[key]
```
Comment (sidk) [inline app/lib/goal_form.rb]: change to an exception

File: app/views/landing/show.html.erb [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
             <p> Use this tool to get help with creating a draft plan, explore best practices in the reference library, or browse the benchmarks. </p>
 
-            <button class="btn get-started-button border-black fg-black rounded-pill">Get Started</button>
+            <%= render 'application/button', content: "Get Started" %>
```
Comment (sidk) [inline app/views/landing/show.html.erb]: change to button_tag

File: bin/generate-fixtures [hunk 3]
```diff
@@ -8,8 +8,97 @@ Benchmarks_workbook =
...
+      jee1_cap: load_cell(cells[3]),
+      jee1_ind: load_cell(cells[4]),
+      spar_cap: load_cell(cells[5]),
+      spar_ind: load_cell(cells[6]),
+      bench_cap: load_cell(cells[7]),
+      bench_ind: load_cell(cells[8]),
+      bench_text: load_cell(cells[9])
+    }
+
+    # {
```
Comment (sidk) [inline bin/generate-fixtures]: delete comments

File: bin/generate-fixtures [hunk 4]
```diff
@@ -8,8 +8,97 @@ Benchmarks_workbook =
...
+          raise "#{jee1_id} is not in the data dictionary (row #{row})"
+        end
+      end
+      if row_[:jee2_cap] && row_[:jee2_ind]
+        jee2_id = "jee2_ind_#{row_[:jee2_cap].downcase}#{row_[:jee2_ind]}"
+        unless data_dictionary[jee2_id]
+          raise "#{jee2_id} is not in the data dictionary (row #{row})"
+        end
+      end
+      # if row_[:spar_cap] && row_[:spar_ind]
```
Comment (sidk) [inline bin/generate-fixtures]: delete comment

File: config/application.rb [hunk 5]
```diff
@@ -10,6 +10,7 @@ module Src
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.2
+    config.autoload_paths << Rails.root.join('lib')
```
Comment (sidk) [inline config/application.rb]: i don't think we need this.

File: package-lock.json [hunk 6]
```diff
@@ -0,0 +1,350 @@
+{
```
Comment (sidk) [inline package-lock.json]: this file can be deleted

File: app/lib/goal_form.rb (hunk 1 above)
Comment (savannidgerinel) [reply app/lib/goal_form.rb]: Done

File: app/views/landing/show.html.erb (hunk 2 above)
Comment (savannidgerinel) [reply app/views/landing/show.html.erb]: Reverted to master

File: bin/generate-fixtures (hunk 3 above)
Comment (savannidgerinel) [reply bin/generate-fixtures]: Done

File: bin/generate-fixtures (hunk 4 above)
Comment (savannidgerinel) [reply bin/generate-fixtures]: Done

File: package-lock.json (hunk 6 above)
Comment (savannidgerinel) [reply package-lock.json]: done

File: config/application.rb (hunk 5 above)
Comment (savannidgerinel) [reply config/application.rb]: removed

**Pull request merged**